content/trash.md

```diff
@@ -4,6 +4,91 @@ metaTitle: "sensenet - Trash"
 metaDescription: "sensenet Trash"
 ---
 
+# introduction
+sensenet has a trash feature. When you delete a document or a folder, and the trash feature is enabled (enabled by default), the content will be wrapped into a special container called the Trash Bag, and will be moved to the Trash Bin (/Root/Trash). 
+
+The Trash Bin is a central place for deleted documents where you can browse, restore or delete content permanently. It is also possible to provide a local (workspace-level) trash for users.
+
+The size of the trash and the time frame while the contents can be recovered from the trash is set by the administrator. Trash feature can be switched on or off globally.
+
+# Delete operation capabilities
+Delete operation allows users to transfer content (documents, folders or whole workspaces) into the Trash Bin - or delete them permanently.
+
+Contents can be deleted through API calls, but these actions are also available on the admin ui:
+
+You can access delete action for basically every content in the doclib by clicking context menu (...) or right click on item.
+![delete action](./img/delete_contextmenu.png)
+
+
+By default configuration, a confirmation popup appears, displaying content name and path, with an additional option to permanently delete the content (skipping trash). 
+![confirm delete](./img/confirm.png)
+
+Deleted (trashed) items can be found in a specific folder having some extra parameter like retention time, quota for size and number of content. These limits can be customized.
+![trash](./img/trasheditem.png)
+
+Actions can also be performed on a trashed item. It can be moved to a specific location (restore from trash), deleted, edited and so on.
```

Comment:
Somewhere here I would mention that content from trash could be restored to a container chosen by the user. By defatult this container is the original parent of the deleted content, but the restore target could be chosen freely, keeping in mind that the type of the restored content should be allowed on the chosen container.